src/Test/IntegrationCase.php

```diff
@@ -93,11 +93,6 @@ public function __construct(
         $this->inputCode = $inputCode;
     }
 
-    public static function create()
```

Comment:
i was thinking about it. yes and not. you cannot use this method. anyway. it will crash after calling it